client/src/pages/Courses/index.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect, useContext } from "react";
+import {
+  Collapse,
+  Card,
+  CardContent,
+  Typography,
+  Button,
+} from "@material-ui/core";
+import AddCircleIcon from "@material-ui/icons/AddCircle";
+import { makeStyles } from "@material-ui/core/styles";
+import { UserContext } from "utils/auth";
+import { getCourses, CourseData } from "utils/courses";
+import Navbar from "components/Navbar";
+import AddCourseStudent from "./AddCourseStudent";
+import AddCourseProf from "./AddCourseProf";
+
+const useStyles = makeStyles({
+  root: {
+    justifyContent: "center",
+    alignItems: "center",
+    textAlign: "center",
+    backgroundColor: "#eceef8",
+  },
+  button: {
+    margin: "20px",
+  },
+  icon: {
+    margin: "5px",
+    marginLeft: "8px",
+    marginRight: "2px",
+  },
+  search: {
+    marginTop: "20px",
+  },
+  addCard: {
+    minWidth: 300,
+    maxWidth: 300,
+    margin: "20px",
+    marginTop: 0,
+    padding: "20px",
+    display: "inline-block",
+  },
+  card: {
+    minWidth: 300,
+    maxWidth: 300,
+    margin: "20px",
+    display: "inline-block",
+  },
+  title: {
+    marginTop: "20px",
+  },
+});
+
+type ClassProps = {
```

Comment:
Looks great!! 👌  You can switch this `type` to an `interface` since the other components are using that format.
i.e.
```typescript
interface ClassProps {
  ...
}
```